content/09.evolution.md

```diff
@@ -2,26 +2,59 @@
-Nevertheless, some fragments between SARS-CoV-2 and RATG13 differ by up to 17%, suggesting a complex natural selection process during zoonotic transfer.
-While the _S_ region is highly similar to that of viruses found in pangolins [@doi:10.1101/2020.02.19.950253], there is no consensus about the origin of _S_ in SARS-CoV-2, as it could potentially be the result either of recombination or coevolution [@doi:10.1093/nsr/nwaa036; @doi:10.1126/sciadv.abb9153].
-Though the intermediate host serving as the source for the zoonotic introduction of SARS-CoV-2 to human populations has not yet been identified, the SARS-CoV-2 virus has been placed within the coronavirus phylogeny through comparative genomic analyses.
-Genomic analyses and comparisons to other known coronaviruses suggest that SARS-CoV-2 is unlikely to have originated in a laboratory -- either purposely engineered and released, or escaped -- and instead evolved naturally in an animal host [@doi:10.1038/s41591-020-0820-9].
-Indeed, the World Health Organization (WHO) have published their intentions to thoroughly investigate the origins of SARS-CoV-2 [@url:https://www.who.int/docs/default-source/coronaviruse/20200802-tors-chn-and-who-agreed-final-version.pdf?sfvrsn=5323728d_2&download=true].
-While the position of the SARS-CoV-2 virus within the coronavirus phylogeny has been largely resolved, the functional consequences of molecular variation between this virus and other viruses, such as its bat and pangolin sister taxa or SARS-CoV-1, remain unknown [@doi:10.1038/s41591-020-0820-9, @doi:10.1038/s41586-020-2012-7].
-Fortunately, the basic genome structure of coronaviruses is highly conserved, and insight into the mechanisms the virus uses to enter cells, replicate, and spread is available from prior research on coronaviruses, which has been instrumental in the mobilization of global research to understand the biology of SARS-CoV-2. 
-
-Additionally, worldwide sequencing of viral samples has provided some preliminary insights into possible mechanisms of adaptation in the virus and the detection of novel variants, and omics-based analysis of patient samples has elucidated some of the biological changes the virus induces in its human hosts.
+The emergence of what is now known to be the pathogen SARS-CoV-2 has dramatically reshaped modern life for the past two years.
+The genomic revolution provided the tools needed to understand the virus in ways that were not feasible during previous pandemics.
+For example, the first genome sequence of the pathogen was released on January 3, 2020, providing valuble information about the pathogen within a month and a half of the first known cases.
+As the pandemic has unfolded, evolutionary questions and methods of investigation have framed the scientific approach to understanding the virus.
+These questions have evolved along with the pandemic.
+Thus far, three major evolutionary questions have emerged.
```

Comment:
```suggestion
Thus far, four major evolutionary questions have emerged.<!--The Who does it affect question will be one we have to be careful to avoid overlap with pathogenesis paper? Also is there a fifth question of what is next? We have never deeply sequenced a virus like this before... viral surveillance that has never been achieved. Implications for preventing future pandemics?-->
```

I can definitely count.